### ISSUE TITLE
Real Time Cell updates in Excel make working in excel impossible

### ISSUE THREAD

**Issue by Ashish424** [ZoomerAnalytics/xlwings#965]
#### I want to build a real time updating tickers for financial data in excel.The code below updates and work as expected,but when I click anywhere on the Sheet which is being updated by a python script and try to update that cell with a value/formula,Excel tends to get extremely laggy.

```
import xlwings as xw
import random
def test():
    wb = xw.Book.caller()
    sht = wb.sheets['Sheet2']
    while True:
        sht.range('A1').value = random.random()
        time.sleep(0.1)


```

**Comment by fzumstein** [comment]
try to increase the sleep time

**Comment by Ashish424** [comment]
Sure,that can be done and does improve interactability,but is there any other way to have same delta time and better interaction?In this [doc](https://docs.xlwings.org/en/stable/vba.html),there is mention of OPTIMIZED_CONNECTION = True,is it of any use in this case.I cannot find this setting in the xlwings addin.

**Comment by fzumstein** [comment]
No, that will have no effect and yes, #824 is the same issue. I don't think there's much you can do now as a workaround. We'd need to make the retry wrapper smarter to handle these situations.

**Comment by Ashish424** [comment]
something like sheets['Sheet2'].is_user_working or an event would be very useful allowing you to pause xlwings and excel communication till the user stops updating sheet.

**Comment by marshaul** [comment]
I assume this is the same issue I have where using xlwings to communicate with Excel basically totally hangs if a cell is actively being edited.

Is there any resolution or workaround yet?

**Comment by leonlbc** [comment]
Hi there. Have anyone found a solution for this issue already? Thanks
